lib/monetize_api.ts

```diff
@@ -36,8 +36,8 @@
     export enum Networks {
         IronSource = 'ironSource',
         IronSourceBidding = 'ironSourceBidding',
-        Vungle = 'Vungle',
-        VungleBidding = 'vungleBidding',
+        LiftoffMonetize = 'liftoffMonetize',
+        Vungle = 'liftoffMonetize',
```

Comment:
The 'liftoffMonetize' enum value should be removed as it is not a valid network. Please remove it.